init/pkg/update_pod_status_util.go

```diff
@@ -743,18 +644,16 @@ func ManageVolumesAndUpdateStatus(podName, namespace, clusterName string, restar
 
 	metadata := getNodeMetadata()
 
-	log.Println("INFO: Updating pod status", "podname", podName)
+	logrus.Info("Updating pod status ", "podname=", podName)
 
-	if err := updateStatus(k8sClient, goctx.TODO(), aeroCluster, podName, podImage,
-		metadata, volumes, dirtyVolumes); err != nil {
+	if err := updateStatus(goctx.TODO(), k8sClient, aeroCluster, podName, podImage, metadata, volumes, dirtyVolumes); err != nil {
```

Comment:
There are around 8 params passed to this func and some of them are directly assigned to metadata struct inside that func. Can't we assign such param like volumes, dirtyVolumes, podImage here only and then pass metadata and remaining params to avoid params clutter